proposals/drafts/campaign_finance_filings.rst

```diff
@@ -75,25 +76,35 @@ likely future conflicts with other entities in the Open Civic Data namespace, so
 this proposal doesn't make any attempt to resolve those conflicts since they
 seem ancillary for now.
 
-TODO
-----
-* Make filing activity actions use same pattern as OCD bill actions (@aepton)
-
 Questions to answer
 -------------------
 * How to model Committee Status and Committee Purpose inside of OCD Organization
   models?
+      Just add additional fields for those. Not sure why this was ever a question.
```

Comment:
Similarly, [W3C Registered Organization](https://www.w3.org/TR/vocab-regorg/#ref_rov_orgStatus) has a `orgStatus` field, which we can consider reusing.